lib/message_encryptor.rb

```diff
@@ -20,7 +20,7 @@ def crypt
   end
 
   def secret
-    salt = Rails.application.secrets.secret_key_base
+    salt = Rails.application.secret_key_base
```

Comment:
This method looks up for the secret key base first in the env variable, second in the credentials file and last in the secrets file.
So we can fully move to use ENV vars as done to configure the rest of secrets, and need neither secrets or credentials file.